packages/frontend/pages/strategies.tsx

```diff
@@ -76,15 +96,20 @@ const useStyles = makeStyles((theme) =>
       marginTop: theme.spacing(4),
     },
     overview: {
-      display: 'flex',
+      display: 'grid',
+      [theme.breakpoints.up('md')]: {
+        gridTemplateColumns: '1fr 1fr 1fr',
+      },
+      [theme.breakpoints.down('sm')]: {
+        gridTemplateColumns: '1fr 1fr',
+      },
+      [theme.breakpoints.down('xs')]: {
+        gridTemplateColumns: '1fr',
+      },
+      gridGap: theme.spacing(2),
       columnGap: '20px',
       marginTop: theme.spacing(3),
     },
-    details: {
```

Comment:
Looks like `yarn run build` is failing, preventing the Vercel deployment.

It's because of this deletion here, since you're still using the `details` class in the `CapDetailsComponent`.

Should be a quick fix!

Reply:
You should either add back the `details` class or use a new class title you've created.